CharacterCreation/01_Races.txt

```diff
@@ -165,68 +225,97 @@ can function both underwater and above water. When in the water, the lungs merel
-attacks. 
+== Seadevil ==
 
-        Subraces:
+* +1 DEX
+* -1 FOR
+* -1 CHA
+* Low-light vision.
 
-Seadevil
--1 to base CHA, -1 to base FORT.
-+1 to DEX
-Low-light vision.
 Ability: Bioluminescence - Once per day.[Is an available skill] Persuade a 
 being, sentient or not, that you are trustworthy. Victim must roll a will-save 
 50+(Siren points). Effect is dissipated when victims eyes are covered or lose 
 sight of you. This takes a tremendous amount of effort and you take a -10 
-penalty if tired or bloodied. This ability will temporarily scramble a robot’s 
+penalty if tired or bloodied. This ability will temporarily scramble a robot's 
 visual cortex, but they can adjust rapidly.
 
-Chondrich
--1 to base CHA, +1 to both STR and FOR
-Tough Skinned: Natural Armor protects you in a firefight. +10 armor coverage, 
-+ 5+(FOR/2) Damage reduction. Armor piercing rounds ignore these bonuses. 
-Large: You are larger than your counterparts and are thus easier to hit, -5 DC 
-on any chance to hit you.
-
-Mendar
-+2 to base CHA
--2 to chosen stats.
-Ability: Colorful! - People like you, they naturally flock toward your brilliant
-scales. You have the ability to either wow or calm a crowd. You are the perfect
-distraction; bring in the attention of those in the area. Victims must roll a 
-will-save of 40+Siren Points. Droids are immune to this effect.
-
-Tetra
--1 to DEX, -1 to STR
-+1 FOR
-Ability: Burst of Speed - [Once per encounter] You are not fast, except when you
-or your schoolmates are in danger. Moving faster than many think possible, you 
-dart forward, moving at 2*DEX for one action. Max 2 Actions. At the end of the 
-dash, take a free melee strike at an enemy within range.
-Ability: Toxic Poof - Your particularly stretchy scales allow for a rapid 
-expansion of your body, protruding the toxic spines that otherwise lay 
-harmlessly under your scales. You expand to twice your size and anything that 
-touches you takes 10 dmg +5 poison dmg per turn until cleansed. If you barrel 
-into anything while poofed, it takes 30 damage +10 poison damage per turn and 
-must roll a shock save DC of 50 to not be paralyzed. You are easier to hit while
-in this state and take an additional %10 damage from any damage done to you.
+== Chondrich ==
+
+* +1 STR
+* +1 FOR
+* -1 CHA
+
+Passive: Tough Skinned 
+
+    Natural Armor protects you in a firefight. +10 armor coverage, + 5+(FOR/2) 
+Damage reduction. Armor piercing rounds ignore these bonuses. 
+
+Passive: Large
+
+    You are larger than your counterparts and are thus easier to hit, -5 DC on 
+any chance to hit you.
+
+== Mendar ==
+
+* +2 CHA
+* -2 to chosen stats.
+
+Ability: Colorful!
+
+    People like you, they naturally flock toward your brilliant scales. You have 
+the ability to either wow or calm a crowd. You are the perfect distraction; 
+bring in the attention of those in the area. Victims must roll a will-save of 
+40+Siren Points. Droids are immune to this effect.

```

Comment:
Are "Siren Points" defined anywhere? I haven't seen them.

I consider several of the Opaleite subraces to be quite underpowered. But I'll let it go since the racial rework is in progress and I can bring up any concerns there.

Reply:
The way I read it, I think Siren Points are Skill Points in a Siren Skill.

Reply:
I c